dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -92,7 +93,13 @@ object Datadog {
         clientToken: String,
         endpointUrl: String? = null
     ) {
-        check(!initialized) { "Datadog has already been initialized." }
+        if (initialized) {
```

Comment:
That's cool. Have one question though...if the user will want to use a logger without initialising the SDK this will still crash at runtime right ? Maybe we need to look into this also.

Reply:
Yes it'll crash. WHy would they import datadog and not initialise it ?

Reply:
Not sure...just thinking about it...I will ask Daniel to tell us also his opinion.